src/pack/ekat_pack_utils.hpp

```diff
@@ -104,6 +104,16 @@ struct PackInfo {
   }
 };
 
+template <typename rngAlg, typename PDF, typename ScalarType, int N>
+void genRandArray(Pack<ScalarType,N> *const x, int length, rngAlg &engine, PDF &&pdf) {
```

Comment:
Should this be snake case (`gen_rand_array`)?

Reply:
We don't have an official formatting rule in ekat. I do prefer snake case, tbh, but I feel like maybe this PR has enough?